src/objects.js

```diff
@@ -1,41 +1,64 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
-};
+  if (object.hasOwnProperty(property)){
```

Comment:
Here could be the same logic as it was in numbers:
object.hasOwnProperty(property) will already return true or false, meaning you could just go:
`return object.hasOwnProperty(property)`